src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1687,6 +1690,34 @@ public String getVariableNamePattern() {
 					})
 					.serializer(new EnumSerializer<>(Gene.class)));
 		}
+		EnumUtils<EntityRegainHealthEvent.RegainReason> regainReasons = new EnumUtils<>(EntityRegainHealthEvent.RegainReason.class, "heal reasons");
+		Classes.registerClass(new ClassInfo<>(EntityRegainHealthEvent.RegainReason.class, "healreason")
+			.user("(regen|heal) (reason|cause)")
+			.name("Heal Reason")
+			.description("The heal reason in a heal event")
+			.usage(regainReasons.getAllNames())
+			.examples("").since("INSERT VERSION").parser(new Parser<EntityRegainHealthEvent.RegainReason>() {
```

Comment:
A stylistic change, but can you separate these three calls on their own line?

Reply:
Done 👍